src/templates/article.js

```diff
@@ -218,7 +219,7 @@ const Article = props => {
                     <ArticleContent>
                         <DocumentBody
                             pageNodes={contentAST}
-                            slugTitleMapping={slugTitleMapping}
+                            //slugTitleMapping={slugTitleMapping}
```

Comment:
We can also remove this from the `ArticleSeries` component and `ComponentFactory`